packages/trie/README.md

```diff
@@ -173,11 +189,19 @@ async function test() {
 test()
 ```
 
-Additional examples with detailed explanations are available [here](https://github.com/gabrocheleau/merkle-patricia-trees-examples).
+Additional examples with detailed explanations are available [here](https://github.com/gabrocheleau/merkle-patricia-trees-examples) (might be incompatible with v5+).
```

Comment:
Ah yeah, I should take the time and update this at some point

Reply:
👍